<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EqualizerDialog</class>
 <widget class="QWidget" name="EqualizerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Equalizer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="toggleEqualizer">
       <property name="text">
        <string>Activate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="presetLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Preset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="presetList">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>104</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="movement">
        <enum>QListView::Static</enum>
       </property>
       <property name="flow">
        <enum>QListView::TopToBottom</enum>
       </property>
       <property name="gridSize">
        <size>
         <width>100</width>
         <height>80</height>
        </size>
       </property>
       <property name="viewMode">
        <enum>QListView::IconMode</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxEqualizer">
     <property name="title">
      <string>Equalizer</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,10">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_preamp">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Preamp</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="MiamSlider" name="preamp_slider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>-200</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="pageStep">
           <number>20</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="preamp_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">0.0 db</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="2">
         <widget class="QLabel" name="band_02_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">0.0 db</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QLabel" name="band_05_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">0.0 db</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="band_03_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">0.0 db</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLabel" name="band_04_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">0.0 db</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <widget class="QLabel" name="band_06_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">0.0 db</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="9">
         <widget class="QLabel" name="band_09_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">0.0 db</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="band_01_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">0.0 db</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="7">
         <widget class="QLabel" name="band_07_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">0.0 db</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="8">
         <widget class="QLabel" name="band_08_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">0.0 db</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="10">
         <widget class="QLabel" name="band_10_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">0.0 db</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="MiamSlider" name="band_04_slider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>-200</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="pageStep">
           <number>20</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="MiamSlider" name="band_01_slider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>-200</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="pageStep">
           <number>20</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="MiamSlider" name="band_03_slider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>-200</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="pageStep">
           <number>20</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="MiamSlider" name="band_06_slider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>-200</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="pageStep">
           <number>20</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="MiamSlider" name="band_02_slider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>-200</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="pageStep">
           <number>20</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="MiamSlider" name="band_05_slider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>-200</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="pageStep">
           <number>20</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="8">
         <widget class="MiamSlider" name="band_08_slider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>-200</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="pageStep">
           <number>20</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="10">
         <widget class="MiamSlider" name="band_10_slider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>-200</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="9">
         <widget class="MiamSlider" name="band_09_slider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>-200</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="7">
         <widget class="MiamSlider" name="band_07_slider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>-200</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="pageStep">
           <number>20</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_band_02">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">170 Hz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_band_01">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">60 Hz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="8">
         <widget class="QLabel" name="label_band_08">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">12 KHz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_band_03">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">310 Hz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLabel" name="label_band_05">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">1 KHz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label_band_04">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">600 Hz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="9">
         <widget class="QLabel" name="label_band_09">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">14 KHz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QLabel" name="label_band_07">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">6 KHz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="10">
         <widget class="QLabel" name="label_band_10">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">16 KHz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QLabel" name="label_band_06">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">3 KHz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MiamSlider</class>
   <extends>QSlider</extends>
   <header>styling/miamslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
